### ISSUE TITLE
Installation on OSX with Ubuntu 19.10

### ISSUE THREAD

**Issue by C-driiic** [ebemunk/phoenix#3]
Hi @ebemunk ,

I'm trying (for 3 days now) to install phoenix on OSX with VirtualBox and Ubuntu 19.10 but without any success 😞 
I'm not a linux professional so I get quickly confused when errors arise. I can see it's been almost two years without any new commit so maybe the latest version of Ubuntu can't handle it ... 

Below are the steps I have followed:
- Modified the Makefile to comment out `WIN=1`
- I have converted the script _ubuntu.sh_ with _dos2unix_ package to avoid errors with DOS line ending `^M`
- then when I did `sudo bash ubuntu.sh` I had issue with the command lines `mkdir release cd release` which was creating, for some reason, the _release_ folder outside the _opencv-2.4.9_ folder. 
- so I manually created the _release_ folder and ran the next `cmake ...` lines

Then errors arised. Saying that some files were missing like _sys/videoio.h_ , _linux/videodev.h_ , etc...
It looks like they are Ubuntu files but I'm not sure and I don't know how to fix that !

I found some forums talking about installing the last _libjasper-dev_ build but I tried, and still no success.

Could you please give me a hand on this. Thanks in advance. Cheers
Cédric

**Comment by ebemunk** [comment]
Hey @Cdik , were you able to figure out your issue?
Have you tried just running `install_scripts/ubuntu.sh` normally without any preprocessing? I regularly provision ubuntu machines with this script and have not had any issues.
If you use the script you don't need to clone the repo or comment out stuff in the Makefile either.

**Comment by C-driiic** [comment]
Hey @ebemunk ,

Yes I managed. Thanks :)
I downloaded Ubuntu 16.04 and followed the instructions in the ubuntu.sh
script step by step but didn't use it directly as is.

I will try to port it to Ubuntu 19.10 and update openCV and Boost at the
same time. I almost managed. There are some namespace changes from openCV 2
to openCV 3.
So far the script built correctly with some adjustments but the new
imread() with IMREAD_COLOR instead of CV_LOAD_IMAGE_COLOR command is
not working and gives me an error at first line when functions.cpp is
trying to read the image.

I need to investigate more on that. It might be because of a conflict with
opencv2 and opencv4 on Ubuntu but not sure.
